code/modules/events/queen_mothers_plasma_boost.dm

```diff
@@ -0,0 +1,31 @@
+/datum/round_event_control/queen_mothers_plasma_boost
+	name = "Queen Mother's Plasma Boost"
+	typepath = /datum/round_event/queen_mothers_plasma_boost
+	weight = 7
+	earliest_start = 45 MINUTES
+	max_occurrences = 2
+
+	gamemode_blacklist = list("Combat Patrol","Civil War","Sensor Capture")
+
+/datum/round_event_control/queen_mothers_plasma_boost/can_spawn_event(players_amt, gamemode)
+	if(length(GLOB.alive_xeno_list) <= 8)
+		return FALSE
+	return ..()
+
+/datum/round_event/queen_mothers_plasma_boost/start()
+	xeno_message("The Queen Mother has refilled the empty plasma stores of our hive. She expects great things, do not fail her.")
+	priority_announce("Sensors have detected a malevolent psychic force hovering over the battlefield, surely this doesn't bode well for us...")
```

Comment:
More fluff? We use TGMC intel division for King pod updates for example
+ the "doesnt bode well" is cheesy and doesnt really explain what happpened